arches/app/media/js/views/components/widgets/urldatatype.js

```diff
@@ -11,6 +11,18 @@ define([
          
         WidgetViewModel.apply(this, [params]);
 
+
+        // #10027 assign this.url & this.url_label with value versions for updating UI with edits
+        if (typeof (this.value) === "function") {
+            if (this.value()) {
```

Comment:
Could just be
```suggestion
        if (ko.isObservable(this.value) && this.value()) {
```

Reply:
I much prefer this.  Applied in [17bfad7](https://github.com/archesproject/arches/pull/10642/commits/17bfad73e18a6dce480859f3a677a4cf6b7e2171).